book/src/example.md

```diff
@@ -1,493 +1,12 @@
 # Example
 
-> ## *This page is out-of-date and under active development.*
+The [Hostpath CSI driver](https://github.com/kubernetes-csi/csi-driver-host-path) is a simple sample driver that provisions a directory on the host. It can be used as an example to get started writing a driver, however it is not meant for production use.
+The [deployment example](https://github.com/kubernetes-csi/csi-driver-host-path/tree/master/deploy) shows how to deploy and use that driver in Kubernetes.
```

Comment:
Sorry, only thought of this now: https://github.com/kubernetes-csi/csi-driver-host-path#deployment is a better link, because it has some actual documentation.

Reply:
good idea! pointing to deployment chapter is more user-friendly than pointer to directory